### ISSUE TITLE
Studio stuck on splash

### ISSUE THREAD

**Issue by efa** [vinegarhq/vinegar#461]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Debian 12
installed with:
$ sudo apt install flatpak
$ flatpak install flathub org.vinegarhq.Vinegar
$ flatpak run org.vinegarhq.Vinegar

launching Studio with:
$ flatpak run org.vinegarhq.Vinegar studio run
its stuck on splash screen forever, no error is show in console.
Attached the log, with CTRL+C and the end to terminate.
[Studio-2024-11-24T22:12:07+01:00.log.tar.gz](https://github.com/user-attachments/files/17894980/Studio-2024-11-24T22.12.07%2B01.00.log.tar.gz)

I tryed to edit the config with:
$ flatpak run org.vinegarhq.Vinegar edit
and set:

```
[splash]
enabled = false

[studio]
#gpu = "prime-discrete"
#gpu = "integrated"
dxvk = false
#dxvk_version = "2.3"
renderer = "D3D11"
#renderer = "OpenGL"
#renderer = "Vulkan"
forced_version = "version-ea7c4a2d814e4dc6"
```

changing some options with no luck

Using:
$ flatpak run org.vinegarhq.Vinegar studio exec winecfg
I set DPI to 97, same results

Note: on the same system Roblox work well, installed with:
$ flatpak install --user https://sober.vinegarhq.org/sober.flatpakref
run with:
$ flatpak run org.vinegarhq.Sober

Note: Flatpak is really a mess, 1 GB of installation for Roblox only and the needed runtime, really not integrated with native distro packaging, very unconfortable

### System information

```shell
Vinegar: v1.7.7 
* Distro: Freedesktop SDK 23.08 (Flatpak runtime)
* Processor: AMD Ryzen 5 5600G with Radeon Graphics
  * Supports AVX: true
  * Supports split lock detection: false
* Kernel: 6.1.0-25-amd64
* Wine (Player): wine-9.18 (Staging)
* Wine (Studio): wine-9.18 (Staging)
* Flatpak: [x]
* Cards:
  * Card 0: amdgpu 0000:08:00.0 /sys/class/drm/card0
```

**Comment by bruhcedric** [comment]
What happens if you try:

```toml
[studio]
forced_version="version-9b68669a3b644e31"
```

in the config?

**Comment by efa** [comment]
work thank you

**Comment by efa** [comment]
What Is the relation between Roblox versione like 2.652.765 and the hash to put in forced_version? In other words, what version Is this that work?

**Comment by bruhcedric** [comment]
> What Is the relation between Roblox version like 2.652.765 and the hash to put in forced_version? In other words, what version Is this that work?

No clue, but I found the solution here: https://github.com/vinegarhq/vinegar/issues/454

**Comment by bruhcedric** [comment]
> work thank you

By the way, are you able to actually get into an actual project? Mine freezes when it's loading an experience.. can't get it to work

**Comment by efa** [comment]
to me work well with loading an experience

**Comment by mrfiguy** [comment]
> What happens if you try:
> 
> ```toml
> [studio]
> forced_version="version-9b68669a3b644e31"
> ```
> 
> in the config?
> 
> I managed to get past the issue where I was stuck on splash by doing this, now I'm stuck on "Loading Studio..." though. (By the way, for me there was a window that appeared behind the splash that told me that I was running an old studio version, and that I had to update to continue.)

For me this version is outdated, if the priblem oc the splash screen occurs again, find a working version at https://setup.rbxcdn.com/DeployHistory.txt

**Comment by bruhcedric** [comment]
> > What happens if you try:
> > ```toml
> > [studio]
> > forced_version="version-9b68669a3b644e31"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > in the config?
> > I managed to get past the issue where I was stuck on splash by doing this, now I'm stuck on "Loading Studio..." though. (By the way, for me there was a window that appeared behind the splash that told me that I was running an old studio version, and that I had to update to continue.)
> 
> For me this version is outdated, if the problem of the splash screen occurs again, find a working version at https://setup.rbxcdn.com/DeployHistory.txt

Yeah, just tried to launch Studio, which did work 2 days ago or so, with the version you mentioned: "version-9b68669a3b644e31". Doesn't work anymore though..
You know a working one?

**Comment by apprehensions** [comment]
An issue with your installed Wine, this issue occured in Flatpak, which will be fixed.
